tools/version-bump.js

```diff
@@ -18,9 +18,31 @@ const RUST_PACKAGE = path.join(__dirname, '../ironfish-rust-nodejs/package.json'
 const SIMULATOR_PACKAGE = path.join(__dirname, '../simulator/package.json')
 
 const shouldBumpIronfishRust = process.argv.find((a) => a.includes('rust'))
+const isPatchRelease = process.argv.find((a) => a.includes('patch'))
+const isMajorRelease = process.argv.find((a) => a.includes('major'))
+
+const bumpVersion = (version) => {
+  if (isMajorRelease) {
+    return bumpMajor(version)
+  } else if (isPatchRelease) {
+    return bumpPatch(version)
+  } else {
+    return bumpDefault(version)
+  }
+}
+
+const bumpMajor = (version) => {
+  const { major, minor, patch } = parseVersion(version)
+  return `${parseInt(major) + 1}.${minor}.${patch}`
+}
```

Comment:
this doesn't set the others back to 0, should it?

Reply:
Oh right, good catch! Fixed